BasicAPI6/LinkedTables.java

```diff
@@ -0,0 +1,61 @@
+package BasicAPI6;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.TreeMap;
+
+// класс для работы с таблицами связанными с экземплярами Notebook
+public class LinkedTables {
+    private HashMap<String, Map<Integer, String>> tables; // связь между ключами Notebook и связанными таблицами
+
+    LinkedTables(){
+        tables = new HashMap<>();
+    }
+
+    public boolean set(String key, Map<Integer, String> object){
+        if (Notebook.IDKEYS.contains(key)){// если вводимый ключ соответствует связанной таблице
+            this.tables.put(key, object);
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    public Map<Integer, String> get(String key){
+        if (this.tables.containsKey(key)){
+            return this.tables.get(key);
+        } else {
+            return null;
+        }
+    }
+
+    public static Map<Integer, String> loadTable(String fileName){
+        //загрузка данных из связанной таблицы
+        Map<Integer, String> table = new TreeMap<>();
```

Comment:
Why TreeMap instead of HashMap?

Reply:
When an option is selecting to be find out, data from linked table are presented for a user. For example, user selects a color of Notebooks to be find out. When he selects a color, he chooses an ID from the appropriate table. So the table should be sorted out.That's why TreeMap is used.